applications/examples/example_number_input/example_number_input.c

```diff
@@ -0,0 +1,75 @@
+#include "example_number_input.h"
+
+bool example_number_input_custom_event_callback(void* context, uint32_t event) {
+    furi_assert(context);
+    ExampleNumberInput* app = context;
+    return scene_manager_handle_custom_event(app->scene_manager, event);
+}
+
+static ExampleNumberInput* example_number_input_alloc() {
+    ExampleNumberInput* app = malloc(sizeof(ExampleNumberInput));
+    app->gui = furi_record_open(RECORD_GUI);
+    app->notification = furi_record_open(RECORD_NOTIFICATION);
+    app->text_buffer = furi_string_alloc();
+    app->current_number = 5;
+    app->min_value = -128;
+    app->max_value = 2345;
+    furi_string_set(app->text_buffer, "5");
+    //Turn backlight on, believe me this makes testing your app easier
+    notification_message(app->notification, &sequence_display_backlight_on);
```

Comment:
I generally hate it when loading an app via fbt and the backlight is not on. Its hard to see whats on screen without the light and it requires a keypress to activate. If the keypress already does something, you miss the initial screen. 

I'll leave it in there, but comment it out because you don't want it.

Reply:
Ok I just noticed this was recently changed, it now turns on without the command... so yeah, not needed anymore, will remove